v3/plugins/containers/buildah/1.14.0/meta.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: containers
+name: buildah
+version: 1.14.0
+type: Che Plugin
+displayName: Buildah Base Dev Plugin
+title: Buildah Base Dev Plugin
+description: Container to build docker images with buildah
+icon: https://raw.githubusercontent.com/containers/buildah/v1.14.0/logos/buildah-logomark_large.png
+repository: https://github.com/containers/buildah
+firstPublicationDate: "2020-04-15"
+category: Other
+spec:
+  containers:
+   - name: buildah-dev
+     image: "quay.io/eclipse/che-buildah-base:1.14.0-b0c91d3"
+     mountSources: true
+     memoryLimit: "3Gi"
```

Comment:
Is `3Gi` really the requirement here? Setting this as a default could cause a lot of issues in adding it to a workspace.

Reply:
I totally agree with you. The value is changed to 1gb, build checked on image for kubernetes plugin sidecar.